prince-of-versions/src/main/java/co/infinum/princeofversions/UpdaterResult.java

```diff
@@ -102,19 +102,19 @@ public boolean isCancelled() {
     @Override
     public void notifyMandatoryUpdate(String version) {
         isConsumed = true;
-        callback.onNewUpdate(version, true);
+        callback.onNewUpdate(version, true, null);
```

Comment:
I'm not sure that you understood what I wanted here. Instead of null, we should pass `Map<String,String>` object which was parsed in `JsonVersionConfigParser `. If the library parses the map and does not pass it to the client there is no point in having this field in a JSON response.

We can talk tomorrow in person and I'll try to clarify it.